ludwig/utils/entmax/activations.py

```diff
@@ -217,11 +226,15 @@ def sparsemax(X, dim=-1, k=None):
     P : torch tensor, same shape as X
         The projection result, such that P.sum(dim=dim) == 1 elementwise.
     """
-
+    # Avoids call to custom autograd.Function during eval to ensure torchscript compatibility
+    # custom autograd.Function is not scriptable: https://github.com/pytorch/pytorch/issues/22329#issuecomment-506608053
+    if not training:
+        output, _ = _sparsemax_forward(X, dim, k)
```

Comment:
I imagine this means we cannot use integrated gradients with torchscript, since we lose the grad info at predict time with this approach, is that right? Not the end of the world, but something we'll need to think about.